tests/test_pre.py

```diff
@@ -59,11 +59,11 @@ def test_illumination_median(image_files):
                                              quantile=0.5,
                                              stretch_quantile=1e-7,
                                              method='median')
-    illum_true = np.array([[ 4.,  5.,  4.,  1.,  1.],
-                           [ 4.,  6.,  2.,  4.,  2.],
-                           [ 8.,  3.,  4.,  2.,  7.],
-                           [ 3.,  3.,  3.,  7.,  6.],
-                           [ 3.,  3.,  3.,  7.,  7.]]) / 10
+    illum_true = np.array([[4., 5., 6., 2., 2.],
+                           [5., 5., 2., 6., 2.],
+                           [4., 3., 6., 4., 7.],
+                           [3., 3., 3., 7., 7.],
+                           [4., 3., 2., 6., 6.]]) / 10
```

Comment:
Manually as in.. loaded ``i``, ``j`` and ``k`` into ipython myself and done the averaging, padding, filtering, etc. myself.. yes, that's how I got these values.